File: test/statuses_test.exs
defmodule StatusesTest do
  use ExUnit.Case
  doctest Statuses

  test "get_statuses returns the list of statuses" do
    list = Statuses.get_statuses()
    # sample status we know is in the list
    sample = %Status{
      desc:
        "A person verified by a 3rd party OAuth provider or by confirming their email address",
      code: 1,
      text: :verified
    }

    assert Enum.member?(list, sample)
  end
end

File: test/statuses_test.dart
import 'package:statuses/statuses.dart';
import 'package:test/test.dart';

void main() {
    test('Fetching statuses', () async {
      final statusesArray = await Statuses.list();
      expect(statusesArray.length, greaterThan(0));
      expect(statusesArray.first.code, 1);
    });
}
